### PR TITLE
Task[#38827] Tạo footer cho trang home và footer chung

### PR DESCRIPTION
**Pull request by tainguyen1612**
https://edu-redmine.sun-asterisk.vn/issues/38827

### CODE REVIEW

**Comment by tainguyen1612** [comment]
ready

**Comment by tainguyen1612** [comment]
ready

File: .gitignore [hunk 1]
```diff
@@ -43,6 +43,10 @@ jspm_packages/
 
 assets/
 dist/
+<<<<<<< HEAD
```
Comment (manhnt-0960) [inline .gitignore]: conflict

**Comment by tainguyen1612** [comment]
ready

**Pull request merged**